include/z64actor.h

```diff
@@ -752,6 +752,10 @@ typedef struct NpcInteractInfo {
 #define PARAMS_PACK(p, s, n) \
     (((p) & NBITS_TO_MASK(n)) << (s))
 
+// Pack all bits in `p` by `s`. There is no masking of specific bits.
```

Comment:
I'm not sure what packing "by s" means. Maybe something like

```c
// Moves the value `p` to bit position `s` for building actor parameters by OR-ing these together.
```

Reply:
the intent was "by s bits" but your suggestion is better anyway, pushed